_posts/en/newsletters/2025-06-27-newsletter.md

```diff
@@ -0,0 +1,159 @@
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo],
+[Lightning BLIPs][blips repo], [Bitcoin Inquisition][bitcoin inquisition
+repo], and [BINANAs][binana repo]._
+
+- [Bitcoin Core #31981][] adds a `checkBlock` method to the inter-process
+  communication (IPC) `Mining` interface (see Newsletter [#310][news310 ipc])
+  that performs the same validity checks as the `getblocktemplate` RPC in
+  `proposal` mode. This enables mining pools using [Stratum v2][topic pooled
+  mining] to validate block templates provided by miners via the faster IPC
+  interface, rather than serialising up to 4 MB of JSON through RPC. The
+  proof-of-work and merkle root checks can be disabled in the options.
+
+- [Eclair #3109][] extends its [attributable failures][topic attributable failures] support (see Newsletter
+  [#356][news356 failures]) to [trampoline payments][topic trampoline payments].
+  A trampoline node now decrypts and stores the part of the attribution payload
+  intended for it and prepares the remaining blob for the next trampoline hop.
+  This PR does not implement the relay of the attribution data for trampoline
+  nodes, which is expected in a follow-up PR.
+
+- [LND #9950][] adds a new `include_auth_proof` flag to the `DescribeGraph`,
+  `GetNodeInfo` and `GetChanInfo` RPCs and to their corresponding `lncli`
+  commands. Including this flag returns the [channel announcement][topic channel
+  announcements] signatures, allowing validation of channel details
+  by third-party software.
+
+- [LDK #3868][] reduces the precision of the [HTLC][topic htlc] hold time for
+  [attributable failure][topic attributable failures] (see Newsletter [#349][news349 attributable]) payloads
+  from 1-millisecond to 100-millisecond units, to mitigate timing-fingerprint
+  leaks. This aligns LDK with the latest updates to the [BOLTs #1044][] draft.
+
+- [LDK #3873][] raises the delay for forgetting a Short Channel Identifier
+  (SCID) after its funding output is spent from 12 to 144 blocks to allow for
+  the propagation of a [splice][topic splicing] update. This is double the
+  72-block delay introduced in [BOLTs #1270][] that was implemented by Eclair
+  (see Newsletter [#359][news359 eclair]). This PR also implements additional
+  changes to the `splice_locked` message exchange process.
+
+- [Libsecp256k1 #1678][] adds a `secp256k1_objs` CMake interface library that
+  exposes all the library’s object files to allow parent projects, such as
+  Bitcoin Core’s planned [libbitcoinkernel][libbitcoinkernel project], to link
+  those objects directly into their own static libraries. This solves the
+  problem of CMake lacking a native mechanism to link static libraries into
+  another and spares downstream users from providing their own `libsecp256k1`
```

Comment:
```suggestion
  problem of CMake lacking a native mechanism to link static libraries together
  and spares downstream users from providing their own `libsecp256k1`
```
I'm not sure this suggestion is correct, but something seems wrong with the way it's written currently.

Reply:
Nothing is raising any flags for me on this PR summary, so Im going to publish as is and we can correct it later if something is off